handout/sections/title.tex

```diff
@@ -2,5 +2,8 @@
 
 \title{Eckstein-Keane-Wolpin models}
 \subtitle{An invitation for transdisciplinary collaboration}
-\author{OSE@Bonn\thanks{Corresponding author: Philipp Eisenhauer, peisenha@uni-bonn.de. We thank  Jano\'s Gabler, Annica Gehlen, Boryana Ilieva, Fedor Ishakov, Lena Janys, Moritz Mendel, Tim Mensinger, Tobias Raabe, Klara R\"ohl, and John Rust for their valuable input. Luis Wardenbach provided outstanding research assistance.}}
+\author{
+	% \includegraphics[height=2.5cm]{../material/crop-ose-logo.pdf} \vspace{-1.8cm} \\ \hspace{6.1cm}

```

Comment:
Good to keep both logos in the material folder, but please deleted commented out line.

Reply:
Only black-white logo included in paper.